hack/update-codegen.sh

```diff
@@ -22,9 +22,11 @@ SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
 # generate the code with:
-${CODEGEN_PKG}/generate-groups.sh \
-	all \
-  github.com/portworx/talisman/pkg/client \
-	github.com/portworx/talisman/pkg/apis \
-  "portworx:v1beta1" \
-  --go-header-file ${SCRIPT_ROOT}/hack/custom-boilerplate.go.txt
+for version in "portworx:v1beta1" "portworx:v1beta2"; do
```

Comment:
There is no need to run this is a loop as the scripts takes multiple space separated packages.
Also this will overwrite the v1beta1 generated code.
We should run pass it like this - 
`generate-groups.sh all github... github.. "portworx:v1beta1,v1beta2"`

Reply:
https://github.com/kubernetes/code-generator/blob/master/generate-groups.sh#L37

Reply:
good catch. fixed.